imaginary.go

```diff
@@ -241,7 +241,7 @@ func parseOrigins(origins string) []*url.URL {
 func memoryRelease(interval int) {
 	ticker := time.NewTicker(time.Duration(interval) * time.Second)
 	go func() {
-		for _ = range ticker.C {
+		for range ticker.C {
```

Comment:
In that case we might have even more merit to bump the version to the big 1.0. I've added the two other versions as an `allow_failures` directive. I could also just remove 'm from the list, not sure what's desired at this point.